site/content/entry/citra-progress-report-2017-september.md

```diff
@@ -0,0 +1,360 @@
++++
+date = "2017-11-01T17:40:00-04:00"
+title = "Citra Progress Report - 2017 September"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Fall arrives once more, and like I mentioned in [August's progress report](/entry/citra-progress-report-2017-august),
```

Comment:
Can we phrase this a little differently we're on our way to the end of fall now? I'm fine with releasing these progress reports late, but the language should at least sound in the present